src/cosmosdb-preview/azext_cosmosdb_preview/tests/latest/test_cosmosdb-InAccountRestore_scenario-Table-Gremlin.py

```diff
@@ -0,0 +1,257 @@
+
+        graph_show = self.cmd('az cosmosdb gremlin graph show -g {rg} -a {acc} -d {db_name} -n {gp_name}').get_output_in_json()
+        assert graph_show["name"] == gp_name
+
+        graph_list = self.cmd('az cosmosdb gremlin graph list -g {rg} -a {acc} -d {db_name}').get_output_in_json()
+        assert len(graph_list) == 1
+
+        assert self.cmd('az cosmosdb gremlin graph exists -g {rg} -a {acc} -d {db_name} -n {gp_name}').get_output_in_json()
+
+        self.cmd('az cosmosdb gremlin graph delete -g {rg} -a {acc} -d {db_name} -n {gp_name} --yes')
+        graph_list = self.cmd('az cosmosdb gremlin graph list -g {rg} -a {acc} -d {db_name}').get_output_in_json()
+        assert len(graph_list) == 0
+
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(name_prefix='cli_test_cosmosdb_gremlin_database_graph_restore')
+    def test_cosmosdb_gremlin_database_graph_restore(self, resource_group):
+        db_name = self.create_random_name(prefix='cli', length=15)
+        gp_name = self.create_random_name(prefix='cli', length=15)
+        partition_key = "/thePartitionKey"
+        default_ttl = 1000
+        new_default_ttl = 2000
+        conflict_resolution_policy = '"{\\"mode\\": \\"lastWriterWins\\", \\"conflictResolutionPath\\": \\"/path\\"}"'
+        indexing = '"{\\"indexingMode\\": \\"consistent\\", \\"automatic\\": true, \\"includedPaths\\": [{\\"path\\": \\"/*\\"}], \\"excludedPaths\\": [{\\"path\\": \\"/headquarters/employees/?\\"}]}"'
+        location = "WestUS"
+
+        self.kwargs.update({
+            'acc': self.create_random_name(prefix='cli', length=15),
+            'db_name': db_name,
+            'gp_name': gp_name,
+            'part': partition_key,
+            'ttl': default_ttl,
+            'nttl': new_default_ttl,
+            'conflict_resolution': conflict_resolution_policy,
+            'indexing': indexing,
+            'loc': location
+        })
+
+        self.cmd('az cosmosdb create -n {acc} -g {rg} --capabilities EnableGremlin --backup-policy-type Continuous --locations regionName={loc}')
+        self.cmd('az cosmosdb gremlin database create -g {rg} -a {acc} -n {db_name}')
+
+        assert not self.cmd('az cosmosdb gremlin graph exists -g {rg} -a {acc} -d {db_name} -n {gp_name}').get_output_in_json()
+
+        graph_create = self.cmd('az cosmosdb gremlin graph create -g {rg} -a {acc} -d {db_name} -n {gp_name} -p {part} --ttl {ttl} \
+            --conflict-resolution-policy {conflict_resolution} --idx {indexing}').get_output_in_json()
+        assert graph_create["name"] == gp_name
+        assert graph_create["resource"]["partitionKey"]["paths"][0] == partition_key
+        assert graph_create["resource"]["defaultTtl"] == default_ttl
+        assert graph_create["resource"]["conflictResolutionPolicy"]["mode"] == "lastWriterWins"
+        assert graph_create["resource"]["indexingPolicy"]["excludedPaths"][0]["path"] == "/headquarters/employees/?"
```

Comment:
same here, `checks` can accept several `self.check()` as an array, you don't have to write so many asserts

Reply:
@ddhamotharan , please comment